lib/recurly/account.php

```diff
@@ -1,5 +1,30 @@
 <?php
-
+/*
+ * Class Recurly_Account
+ * @property string $adjustments The URL of adjustments for the specified account.
+ * @property string $account_balance The URL of the account balance for the specified account.
+ * @property string $billing_info The URL of billing info for the specified account.
+ * @property string $invoices The URL of invoices for the specified account.
+ * @property string $redemption The URL of the coupon redemption for the specified account.
+ * @property string $subscriptions The URL of subscriptions for the specified account.
+ * @property string $transactions The URL of transactions for the specified account.
```

Comment:
I don't think these values that are listing as being URLs will actually be present as strings. I think they'll be Stubs for loading the full object.